src/WebJobs.Script.WebHost/Management/LinuxSpecialization/IRunFromPackageHandler.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.WebHost.Models;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Management.LinuxSpecialization
+{
+    public interface IRunFromPackageHandler
+    {
```

Comment:
Was originally here. but moving out to make testing possible. Ideally would have been in a new interface , but just put in IBashCommandHandler since it seemed related enough :)
But can create a new interface if you think that is better.

Reply:
Moving to new interface.